app/views/sign_and_submit/edit.html.haml

```diff
@@ -11,8 +11,7 @@
     .form-card__content
       %p
         By entering your name you agree that you want to apply for
-        FAB and Medicaid, that you have been honest on this application, and
-        that you have read and agreed to the terms on the previous page.
+        FAB, that you have been honest on this application.
```

Comment:
I assume the user will know what "FAB" is (I do not). If that's in question, maybe we can link this text to somewhere that explains it?